### PR TITLE
Added category stats to leaderboard

### PR DESCRIPTION
**Pull request by MRuy**

### CODE REVIEW

File: src/pages/stats.js [hunk 1]
```diff
@@ -11,6 +14,20 @@ const IndexPage = () => {
...
+      data: []
+    });
+    
+    const categoryStatsTitles = [
+      'Sponsor',
+      'Intro',
+      'Outro',
+      'Interaction',
+      'Selfpromotion',
+      'Music offtopic',
```
Comment (ajayyy) [inline src/pages/stats.js:28]: Can this be changed to the following:

`Self Promotion`
`Non-Music Section`

**Comment by ajayyy** [comment]
What is `ink` used for?

File: src/pages/stats.js [hunk 2]
```diff
@@ -42,12 +66,21 @@ const IndexPage = () => {
     }
 
     useEffect(() => {
-        fetch("https://sponsor.ajay.app/api/getTotalStats")
+        fetch(API_BASE + "/api/getTotalStats")
             .then(response => response.json())
             .then(resultData => setTotalStats(resultData));
 
-        setTopUserData("https://sponsor.ajay.app/api/getTopUsers?sortType=0");
+        setTopUserData(API_BASE + "/api/getTopUsers?sortType=0&categoryStats");
```
Comment (ajayyy) [inline src/pages/stats.js:73]:
```suggestion
        setTopUserData(API_BASE + "/api/getTopUsers?sortType=0&categoryStats=true");
```

**Comment by MRuy** [comment]
I added `ink` because `gatsby` complained about missing it.  
Found out later that the install kind of failed and forgot to remove it.

File: gatsby-config.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ module.exports = {
             resolve: "gatsby-plugin-purgecss",
             options: {
                 printRejected: true,
+                whitelist: ['row--even'],
```
Comment (ajayyy) [inline gatsby-config.js:26]: You can remove purgecss entirely if you'd like. It may help with future issues.

**Comment by ajayyy** [comment]
Could you add a checkbox disabled by default that makes this popup appear. This would allow scrolling through stats without it covering them.

**Pull request merged**